src/styles/WishlistScreen/Screen_Wishlist.jsx

```diff
@@ -0,0 +1,45 @@
+import { StyleSheet, Dimensions } from "react-native";
+import Constants from "expo-constants";
+import { COLORS } from "../colors";
+
+export const containers = StyleSheet.create({
+  bgContainer: {
+    /* marginTop: Constants.statusBarHeight, */
+    backgroundColor: COLORS.PURPLE,
+    height: Dimensions.get("screen").height-170,
+    justifyContent: "space-evenly",
+  },
+  productsContainer: {
+    justifyContent: "center",
+    alignItems: "center",
+    alignContent: "center",
+    alignSelf: "center",
+    marginRight: 40,
+    marginLeft: 40,
+    padding: 10,
+    marginTop: 0,
```

Comment:
If you are using more than two margins it is recommendable to combine them in a single style. In this case, if your marginTop and marginBottom are 0 ,you can change it to `margin: 0 40` and you save some lines.
https://www.w3schools.com/css/css_margin.asp

Reply:
After some tests, I decided to use `marginHorizontal` and `marginVertical`, because it seems that `margin` do not support more than 2 values, but `marginHorizontal` can set the margin for both right and left, like `marginVertical` sets the margin for both top and bottom.